docs/conversion/sakai_20_2-20_3_mysql_conversion.sql

```diff
@@ -19,3 +19,7 @@ ALTER TABLE lti_tools RENAME COLUMN allowsettings TO allowsettings_ext;
 -- UPDATE lti_tools SET allowsettings_ext=allowsettings;
 -- ALTER TABLE lti_tools DROP COLUMN allowsettings;
 -- END SAK-44810
+
+-- SAK-41502
+ALTER TABLE GB_GRADING_EVENT_T ADD IS_EXCLUDED INT DEFAULT null NULL;
```

Comment:
The original PR is slightly different:

`ALTER TABLE GB_GRADING_EVENT_T ADD IS_EXCLUDED INTEGER NOT NULL DEFAULT 0;`

Reply:
I was going off what that code says according to the hbm.

Reply:
If it should be non null and have a default then the code/hbm are incorrect.

Reply:
I asked for it to be non-null and have a default value [here](https://github.com/sakaiproject/sakai/pull/8364#discussion_r454433678) and [here](https://github.com/sakaiproject/sakai-reference/pull/90#pullrequestreview-448192191).